libeantic/e-antic/fmpq_poly_extra.h

```diff
@@ -13,7 +13,7 @@
 #ifndef E_ANTIC_FMPQ_POLY_EXTRA_H
 #define E_ANTIC_FMPQ_POLY_EXTRA_H
 
-#include "local.h"
+#include <e-antic/local.h>
```

Comment:
```suggestion
// Include architecture specific settings from e-antic/local.h and not just local.h since the header might be installed to separate architecture specific location on the search path.
#include <e-antic/local.h>
```

Can we also add a disclaimer like that everywhere? Otherwise, we might just revert this in a few years when we have forgot why this was necessary I am afraid.

Reply:
idem